### DISCUSSION TITLE
C++ support

### DISCUSSION THREAD

**Discussion by arlucio** [vmware-archive/rules_oss_audit#11] [General]
Hello and congratulations to the team for the great project!

I'm really interested in the C++ support as was mentioned that a prototype was being developed. 

Was there any progress on it? Is there any way to see some code and contribute? 
Would be great to have an insight of how it was implemented.

**Comment by martinbrook** [comment]
I would also be interested in any update on this. 

Hi @geekygirldawn , are you involved with this project at all?

**Comment by geekygirldawn** [comment]
@martinbrook - long time, no see! :) 

Let me check with @danielmachlab and see what he knows about this.
